src/sources/dnstap/mod.rs

```diff
@@ -45,8 +46,8 @@ pub struct DnstapConfig {
 
     /// Absolute path to the socket file to read DNSTAP data from.
     ///
-    /// The DNS server must be configured to send its DNSTAP data to this socket file. The socket file will be created,
-    /// if it doesn't already exist, when the source first starts.
+    /// The DNS server must be configured to receive its DNSTAP data from this socket file. The socket file will be created
+    /// if it doesn't already exist when the source first starts.
```

Comment:
I think this change is wrong? Vector would be reading/receiving from this socket - but the DNS server is sending data to it?